docs/apache-airflow/core-concepts/xcoms.rst

```diff
@@ -56,6 +56,31 @@ XComs are a relative of :doc:`variables`, with the main difference being that XC
 
   If the first task run is not succeeded then on every retry task XComs will be cleared to make the task run idempotent.
 
+
```

Comment:
This should be HowTo in provider docs as well.

Reply:
Will do